docs/Database-Structure.md

```diff
@@ -343,7 +343,8 @@ CREATE TABLE "pa_activation_history"
     "activation_status"  INTEGER,
     "event_reason"       VARCHAR(255),
     "external_user_id"   VARCHAR(255),
-    "timestamp_created"  TIMESTAMP (6) NOT NULL
+    "timestamp_created"  TIMESTAMP (6) NOT NULL,
+    "version"  INTEGER
```

Comment:
Tohle chce změnit i v tom obrázku, ale nevím, kde k tomu máte zdroják.

Reply:
The image is generated by IntelliJ IDEA using `Diagrams | Show Visualisations`. I can update the schema if you won't be able to make it work.

Reply:
Ok, I can update schema myself.

Reply:
We'll need to test how the black color will work on our developer portal, the image is displayed on the page:
https://developers.wultra.com/components/powerauth-server/1.2.x/documentation/Database-Structure

Reply:
Oh, it is caused by dark mode in IDE, I did not realize that. I may change it.

Reply:
It is not dark anymore but still little bit different style.